test/integration/mod/variable_names.mod

```diff
@@ -0,0 +1,65 @@
+: Collection of tricky variable names.
+
+NEURON {
+    SUFFIX variable_names
+}
+
+STATE {
+    is
+    be
+    count
+    as
+    def
+    del
+    elif
+    finally
+    import
+    in
+    pass
+    raise
+    yield
+
+    alpha
+    beta
+    gamma
+    delta
+    epsilon
+    zeta
+    eta
+    theta
+    iota
+    kappa
+    lambda
+    mu
+    nu
+    xi
+    omicron
+    : pi
+    chi
+    psi
+    omega
+}
+
+BREAKPOINT {
+    SOLVE state METHOD sparse
+}
+
+KINETIC state {
+    ~ is <-> be (1, 1)
+    ~ as <-> count (1, 1)
+    ~ def <-> del (1, 1)
+    ~ elif <-> finally (1, 1)
+    ~ import <-> in (1, 1)
+    ~ lambda <-> pass (1, 1)
+    ~ raise <-> yield (1, 1)
+    ~ alpha <-> beta (1, 1)
+    ~ gamma <-> delta (1, 1)
+    ~ epsilon <-> zeta (1, 1)
+    ~ eta <-> theta (1, 1)
+    ~ iota <-> kappa (1, 1)
+    ~ lambda <-> mu (1, 1)
+    ~ nu <-> xi (1, 1)
+    ~ omicron <-> pi (1, 1)
```

Comment:
With `pi` commented out above, this doesn't compile under NOCMODL, is that intended (and do we care)?

Reply:
No, that's a bug in NMODL and a mistake in the MOD file. `pi` is `3.14` and is not a valid name of a variable.